client-hints/echo_client_hints_received.py

```diff
@@ -5,6 +5,8 @@ def main(request, response):
     """
 
     response.headers.append("Access-Control-Allow-Origin", "*")
+    response.headers.append("Access-Control-Allow-Headers", "*")
```

Comment:
@clelland this resource is used by http_equiv_accept_ch_lifetime_subresource.tentative.https.html.

Do you think these changes could have made that test flaky?